python/cutlass/op/conv.py

```diff
@@ -733,7 +734,8 @@ def run(self, A=None, B=None, C=None, D=None,
             stride=(1, 1), padding=(0, 0), dilation=(1, 1),
             alpha=None, beta=None,
             split_k=("serial", 1), sync: bool = True,
-            print_module: bool = False) -> Conv2dArguments:
+            print_module: bool = False,
+            stream: cuda.CUstream = cuda.CUstream(0)) -> Conv2dArguments:
```

Comment:
Could you please update the docstrings for this and other methods that now take in `stream` as an argument?

Reply:
Hi Jack, sure! This is addressed in commit 91becf9022ce9235d8af6fe7c4e648e6f7e8cb96.